src/sap/fhir/model/r4/lib/FHIRUrl.js

```diff
@@ -19,8 +19,11 @@ sap.ui.define([], function() {
 	 * @since 1.0.0
 	 * @version ${version}
 	 */
-	var FHIRUrl = function(sUrl, sServiceUrl) {
+	var FHIRUrl = function (sUrl, sServiceUrl) {
 		this._sServiceUrl = sServiceUrl;
+		if (sServiceUrl.indexOf("http") == 0 || sServiceUrl.indexOf("https") == 0) {
```

Comment:
this part is quite server implementation specific. How do we know that the server supports https on nextlinks?
Might be not.
For me the current behavior of the hapi end point is wrong.

Reply:
This is just added to avoid console errors from the fhir model.